### DISCUSSION TITLE
Cannot check/uncheck files in preview window

### DISCUSSION THREAD

**Discussion by AlphaScorpii-dev** [iterate-ch/cyberduck#16452] [Q&A]
Hello folks
I've started using Cyberduck a couple of weeks ago, following a migration to MacOS. Before that I was mostly using WinSCP.
I have a few synchronization tasks to operate on a regular basis, and I thought that Cyberduck could handle them. However, there is a tiny little essential thing that I haven't yet figured out.
When I am synchronizing a folder, I get to choose between 3 modes (upload/download/mirror), which alter the preview window by checking/unchecking some of the files. I assumed this was to be used as a starting point, because in the end I may or may not want to force a few files in whatever direction. However, all checkboxes are grayed out, so I am limited to what Cyberduck wants me to do no matter what.
Am I missing something here? I haven't found either how to display only the changes, but it seems to have been a feature request for quite a while now.

**Comment by AliveDevil** [comment]
On Windows, the interface does allow to change the selection of items in the Transfer-preparation window.
This behavior is imo an issue on the macOS interface side - what's interesting is, that the checkbox for folders works fine, just files are disabled.

**Reply by AlphaScorpii-dev** [reply]
Ok, so it works as intended for windows, not on  macOS. Any insight as for the why of such difference?
